app/controllers/candidate_interface/course_choices/course_decision_controller.rb

```diff
@@ -1,6 +1,8 @@
 module CandidateInterface
   module CourseChoices
     class CourseDecisionController < BaseController
+      include AdviserStatus
```

Comment:
Does it make sense to mix in the `adviser_sign_up` helper like this in the controllers that need it or should we just put it in the `CandidateInterfaceController` so that we don't have to worry about it when adding to new controllers?

Reply:
There's no performance hit. I think @stevehook suggestion of putting it in `CandidateInterfaceController` is sensible.

Reply:
I think it still makes sense to extract a component although it won't reduce the LOC in each template much because each instance needs different content. Probably worth doing if we expect to change more such links otherwise not so much. What do you reckon @frankieroberto ?